build.proj

```diff
@@ -2,6 +2,16 @@
 <Project ToolsVersion="14.0" DefaultTargets="BuildTheWholeCli" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="build/Microsoft.DotNet.Cli.Monikers.props" />
+  <ImportGroup Condition=" '$(GeneratingPropsFile)' == 'true' ">
+    <Import Condition=" Exists('$(GitCommitInfoProps)') "
+            Project="$(GitCommitInfoProps)" />
+    <Import Condition=" Exists('$(HostInfoProps)') "
+            Project="$(HostInfoProps)" />
+  </ImportGroup>
+  <ImportGroup Condition=" '$(GeneratingPropsFile)' != 'true' ">
+    <Import Project="$(GitCommitInfoProps)" />
+    <Import Project="$(HostInfoProps)" />
+  </ImportGroup>
 
```

Comment:
Do you know if I can do an import after PropertyGroup definition? I tried having these in dir.props, but those properties $(GitCommitInfoProps) and $(HostInfoProps) are defined in dir.props